src/test/java/org/mule/extension/sftp/SftpCopyTestCase.java

```diff
@@ -43,7 +42,7 @@ protected String getConfigFile() {
   }
 
   private String getPath(String... path) throws Exception {
-    return normalizePath(Paths.get(testHarness.getWorkingDirectory(), path).toString());
+    return normalizePath(testHarness.getWorkingDirectory() + String.join("/", path));
```

Comment:
why join instead of simply `+`?

Reply:
Since `path` variable is a varargs, I thought this way would be more convenient rather than iterating through the array for concatenating and building the path